### PR TITLE
feat: updated logging for deeplink

### PR DESCRIPTION
**Pull request by wadeking98**

### CODE REVIEW

**Review by jleach — CHANGES_REQUESTED** [review]
You may have checked in `app/ios/AriesBifold.xcodeproj/project.pbxproj` by mistake - no?

File: app/ios/AriesBifold.xcodeproj/project.pbxproj
Comment (bryce-mcmath) [inline app/ios/AriesBifold.xcodeproj/project.pbxproj:1]: I don't think we should commit changes to this file as they are heavily dependent on local dev env

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
One small thing

File: app/ios/AriesBifold.xcodeproj/project.pbxproj
Comment (bryce-mcmath) [reply app/ios/AriesBifold.xcodeproj/project.pbxproj:1]: Oh sorry, Jason already caught this

Comment (wadeking98) [reply app/ios/AriesBifold.xcodeproj/project.pbxproj:1]: woops my bad, I'll revert

**Review by bryce-mcmath — APPROVED** [review]

**Review by jleach — APPROVED** [review]

**Pull request merged**